proto/rill/runtime/v1/queries.proto

```diff
@@ -147,6 +147,10 @@ service QueryService {
   rpc TableRows(TableRowsRequest) returns (TableRowsResponse) {
     option (google.api.http) = {get: "/v1/instances/{instance_id}/queries/rows/tables/{table_name}"};
   }
+
+  rpc CustomQuery(CustomQueryRequest) returns (CustomQueryResponse) {
+    option (google.api.http) = {post: "/v1/instances/{instance_id}/queries/sql"};
+  }
```

Comment:
Can we just update the existing `Query(QueryRequest) returns (QueryResponse)` API instead?